### PR TITLE
Validation for empty property creation for string, number

### PR DESCRIPTION
**Pull request by KirthiNair**

### CODE REVIEW

File: src/browser/modules/EditorInfo/AddProperty.jsx [hunk 1]
```diff
@@ -133,7 +133,11 @@ function AddProperty (props) {
       valueInput = (
         <TextInput
           id='propValue'
-          value={props.properties ? Object.values(props.properties) : null}
+          value={
+            props.properties
+              ? Object.values(props.properties)
+              : myState.newProperties.propValue
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/AddProperty.jsx:155]: why cant use state only

File: src/browser/modules/EditorInfo/AddProperty.jsx [hunk 2]
```diff
@@ -149,7 +149,11 @@ function AddProperty (props) {
       valueInput = (
         <TextInput
           id='propValue'
-          value={valueProperties || null}
+          value={
+            props.ToDisplay
+              ? valueProperties || null
+              : myState.newProperties.propValue
```
Comment (rahul-tarafdar-synerzip) [inline src/browser/modules/EditorInfo/AddProperty.jsx:155]: stay with one state value, either valueProperties or myState.newProperties.propValue

i am not sure with which one to stay.

**Comment by rahul-tarafdar-synerzip** [comment]
close this pr in favour of pr 81

**Comment by KirthiNair** [comment]
closing this pr in favour of pr 81

**Pull request closed without merging**